docs/index.html

```diff
@@ -2,7 +2,7 @@
 <html class="ts-loading">
 	<head>
 		<title>Tradeshift UI</title>
-		<link rel="icon" href="data:;base64,iVBORw0KGgo="/>
+		<link rel="icon" href="https://developers.tradeshift.com/icons/favicon-32x32.png"/>
```

Comment:
We should probably move this icon to be part of our codebase, either base64 or just deploy the file.

Maybe we should have our own icon which is similar to it, but has some sort of ts.ui feel. What do you think @mihneaz ?